DevCycle/DVCClient.swift

```diff
@@ -296,6 +304,11 @@ public class DVCClient {
             return self
         }
         
+        internal func service(_ service: DevCycleServiceProtocol) -> ClientBuilder {
```

Comment:
I think you can create an extension in the test class if we plan to only use this function in test classes: https://docs.swift.org/swift-book/LanguageGuide/Extensions.html

Reply:
huh maybe this isn't possible